<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YJChatClass</class>
 <widget class="QMainWindow" name="YJChatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YJChat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
      <item>
       <widget class="QTextBrowser" name="m_textBrowser"/>
      </item>
      <item>
       <widget class="QTextEdit" name="m_textEdit"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="m_serverIpLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Server IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_serverIpEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_serverIp_okButton">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="m_tableWidget">
        <column>
         <property name="text">
          <string>Username</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Hostname</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>IP Adress</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>268</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_sendButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>178</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="yjchat.qrc"/>
 </resources>
 <connections/>
</ui>
